csrc/ir/utils.cpp

```diff
@@ -752,6 +752,15 @@ bool isSelectInput(TensorView* tv) {
   return false;
 }
 
+bool isSliceInput(TensorView* tv) {
```

Comment:
Let's not add a trivial function like this.

Reply:
merged with `isSelectInput` into a variadic template function.